sdk/communication/communication-phone-numbers/test/internal/headers.spec.ts

```diff
@@ -2,23 +2,24 @@
 // Licensed under the MIT license.
 
 import { AzureKeyCredential } from "@azure/core-auth";
-import { isNode, WebResourceLike } from "@azure/core-http";
+import { isNode } from "@azure/core-util";
 import { TokenCredential } from "@azure/identity";
 import { assert } from "chai";
 import sinon from "sinon";
-import { PhoneNumbersClient } from "../../src/phoneNumbersClient";
+import { PhoneNumbersClient, PhoneNumbersClientOptions } from "../../src/phoneNumbersClient";
 import { getPhoneNumberHttpClient } from "../public/utils/mockHttpClients";
 import { SDK_VERSION } from "../../src/utils/constants";
 import { Context } from "mocha";
 import { createMockToken } from "../public/utils/recordedClient";
+import { PipelineRequest } from "@azure/core-rest-pipeline";
 
 describe("PhoneNumbersClient - headers", function () {
   const endpoint = "https://contoso.spool.azure.local";
   const accessKey = "banana";
   let client = new PhoneNumbersClient(endpoint, new AzureKeyCredential(accessKey), {
     httpClient: getPhoneNumberHttpClient,
-  });
-  let request: WebResourceLike;
+  } as PhoneNumbersClientOptions);
```

Comment:
Curious why a cast is needed here?  `PhoneNumbersClientOptions` should extends from `core-client`'s `CommonClientOptions`.